lib/models/User.js

```diff
@@ -0,0 +1,53 @@
+const pool = require('../utils/pool');
+
+class User {
+  id;
+  email;
+  first_name;
+  last_name;
+  #password_hash;
+
+  constructor(row) {
+    this.id = row.id;
+    this.email = row.email;
+    this.first_name = row.first_name;
+    this.last_name = row.last_name;
+    this.#password_hash = row.password_hash;
+  }
+
+  static async insert({ first_name, last_name, email, password_hash }) {
+    const { rows } = await pool.query(
+      ` 
+      INSERT INTO users (first_name, last_name, email, password_hash)
+            VALUES ($1, $2, $3, $4)
+            RETURNING *
+            `,
+      [first_name, last_name, email, password_hash]
+    );
+    return new User(rows[0]);
+  }
+  static async getByEmail(email) {
+    const { rows } = await pool.query(
+      ` SELECT * 
+    FROM users WHERE email = $1`,
+      [email]
+    );
+    if (!rows[0]) return null;
+    return new User(rows[0]);
+  }
+
+  static async getAll() {
```

Comment:
```User.getAll()``` returns all the rows from the secrets table? why would you do that when you already have ```Secret.getAll()``` that does the exact same thing and sounds like a closer description of what you're doing?